docker/cygnus-ngsi/README.md

```diff
@@ -0,0 +1,190 @@
+time=2016-05-05T10:01:32.050UTC | lvl=ERROR | corr=8bed4f8d-c47f-499a-a70d-365883584ac7 | trans=8bed4f8d-c47f-499a-a70d-365883584ac7 | srv=default | subsrv=/ | function=processNewBatches | comp=cygnus-ngsi | msg=com.telefonica.iot.cygnus.sinks.NGSISink[411] : Persistence error (Communications link failure. The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.)
+time=2016-05-05T10:01:32.051UTC | lvl=INFO | corr=8bed4f8d-c47f-499a-a70d-365883584ac7 | trans=8bed4f8d-c47f-499a-a70d-365883584ac7 | srv=default | subsrv=/ | function=processNewBatches | comp=cygnus-ngsi | msg=com.telefonica.iot.cygnus.sinks.NGSISink[423] : TTL exhausted, finishing internal transaction (8bed4f8d-c47f-499a-a70d-365883584ac7)
+```
+
+Don't worry about the error, it is normal (please, see next section).
+
+You can stop the container as:
+
+```
+$ docker stop 9ce0f09f5676
+9ce0f09f5676
+$ docker ps
+CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+```
+
+[Top](#top)
+
+###<a name="section3.2"></a>Using a specific configuration
+As seen above, the default configuation distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
+
+* It only works for building historical context data in MySQL.
+* The endpoint for MySQL is `mysql`.
+* The logging level is `INFO`.
+* The logging appender is `console`.
+
+You may need to alter the above values with values of your own.
+
+[Top](#top)
+
+####<a name="section3.2.1"></a>Editing the docker files
+The easiest way is by editing both the `Dockerfile` and/or `agent.conf` file under `docker/cygnus-ngsi` and building the cygnus-ngsi image from the scratch.
+
+This gives you total control on the docker image.
+
+[Top](#top)
+
+####<a name="section3.2.2"></a>Environment variables
+Those parameters associated to an environment variable can be easily overwritten in the command line using the `-e` option. For instance, if you want to change the log4j logging level, simply run:
+
+    $ docker run -e LOG_LEVEL='DEBUG' cygnus-ngsi
+
+[Top](#top)
+
+####<a name="section3.2.3"></a>Using volumnes 
+Another possibility is to start a container with a volume (`-v` option) and map the entire configuraton file within the container with a local version of the file:
+
+    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf cygnus-ngsi
+    
+Of course, you can combine volumnes and environment variables overwritting:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/993/commits/59d14b3709656c1f7bde8e2d00776f79de6d1c0a